### DISCUSSION TITLE
Adding new logs via Logstash

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#4593] [Unsupported Versions]
Hello!

I'm on my journey to import DNS logs and have created custom Logstash configuration files that work fine and declared them to the manager node via global.sls in my distributed deployment like this (I followed the the SO documentation):

logstash:
    pipelines:
      manager:
        config:
          - so/0009_input_beats.conf
          - so/0010_input_hhbeats.conf
          - so/9999_output_redis.conf.jinja
          **- custom/dns-gdc-import.conf
          - custom/dns-pdc-import.conf**
      search:
        config:
         - so/0900_input_redis.conf.jinja
         - so/9000_output_zeek.conf.jinja
         - so/9002_output_import.conf.jinja
         - so/9034_output_syslog.conf.jinja
         - so/9100_output_osquery.conf.jinja
         - so/9400_output_suricata.conf.jinja
         - so/9500_output_beats.conf.jinja
         - so/9600_output_ossec.conf.jinja
         - so/9700_output_strelka.conf.jinja
         **- custom/dns-output.conf**

For testing purposes, the dns-output.conf is supposed to write the imported DNS logs just to a directory on the search node. However, the dns-output.conf gets executed on the manager **AND** the search node. So now I have two copies of all my DNS logs on the manager and the  search node. Why? Any help is highly ppreciated! Thanks in advance!

Cheers, Ben

**Comment by weslambert** [comment]
Are you running a managersearch node instead of just a manager? The manager should only be running the `manager` pipeline.

**Reply by ben-sec** [reply]
Hi Wes,

according to the grid information in SOC I'm running a Manager node.

Cheers, Ben

**Comment by ben-sec** [comment]
No more ideas what's going on here?
